launcher/minecraft/PackProfile.h

```diff
@@ -145,7 +145,7 @@ class PackProfile : public QAbstractListModel
     // todo(merged): is this the best approach
     void appendComponent(ComponentPtr component);
 
-    ModAPI::ModLoaderTypes getModLoaders();
+    std::optional<ResourceAPI::ModLoaderTypes> getModLoaders();
```

Comment:
perhaps, but since other value types do not necessarily have a 'Invalid' state, they need to use optionals, and I think using them all consistently is easier to reason about than having some use an invalid state, and some using optionals

Reply:
Good point.